src/extension/iSamples_resultList.js

```diff
@@ -122,9 +125,9 @@ class ResultList extends React.Component {
                   <CesiumMap
                     mapInfo={this.state}
                     setCamera={setView}
-                    searchFields={searchFields}
-                    bbox={bbox}
-                    onChange={onChange} />
+                    SearchFields={searchFields}
+                    Bbox={bbox}
+                    onSetFields={onChange} />
```

Comment:
Curious why you capitalized these variable names?  Typically classnames are capitalized and variables are not.

Reply:
Unless there is a compelling reason to have these capitalized, I recommend you revert these to camel case in the cleanup PR.